content/docs/12.5-vertex/00-fips.md

```diff
@@ -7,21 +7,58 @@ hideToC: false
 fullWidth: false
 ---
 
+# Overview
+
 Palette VerteX is FIPS 140-2 compliant. This means that Palette VerteX uses FIPS 140-2 compliant algorithms and encryption methods. With its additional security scanning capabilities, Palette VerteX is designed to meet the stringent requirements of regulated industries. Palette VerteX operates on FIPS-compliant Ubuntu Pro versions.
 
 
 ## Non-FIPS Enablement
 
-You can deploy non-FIPS-compliant components in your Palette VerteX environment if you enable non-FIPS settings. Refer to the [Enable non-FIPS Settings](/vertex/system-management/enable-non-fips-settings) documentation for more information.
+You can deploy non-FIPS-compliant components in your Palette VerteX environment by enabling non-FIPS settings. Refer to the [Enable non-FIPS Settings](/vertex/system-management/enable-non-fips-settings) guide for more information.
+
+<!-- When Palette VerteX consumes upstream binaries, it displays FIPS status based on the FIPS rating given to the third-party image.   -->
+
+
+Something to note when using RKE2 and K3s:
+
+<br />
+
+<!-- - Palette VerteX uses compiled images directly from Rancher's RKE2 repository. Since some internal RKE2 components may not be FIPS-compliant, Palette displays RKE2 as a partially compliant layer.  -->
+
+- Rancher considers RKE2 to be FIPS-compliant, as described in it's RKE2 [FIPS 140-2 Enablement](https://docs.rke2.io/security/fips_support) security documentation. Therefore, Palette VerteX designates RKE2 as FIPS-compliant. 
+
+  <br />
+  
+  When we scan the binaries, which we consume directly from Rancher's RKE2 repository, issues are reported for the following components.  This is because they are built with the Go Programming Language (Golang) compiler, which is not FIPS-compliant.
+
+  - container-suseconnect
+  - container-suseconnect-zypp
+  - susecloud
+
+  <br />
+
+  Since these components are unrelated to Kubernetes and are instead used to access SUSE’s repositories during the docker build process, RKE2 itself remains fully compliant. We recommend using RKE2 [FIPS 140-2 Enablement](https://docs.rke2.io/security/fips_support) security documentation as the official source of FIPS compliance.
```

Comment:
```suggestion
  Since these components are unrelated to Kubernetes and are instead used to access SUSE’s repositories during the Docker build process, RKE2 itself remains fully compliant. We recommend using RKE2 [FIPS 140-2 Enablement](https://docs.rke2.io/security/fips_support) security documentation as the official source of FIPS compliance.
```